tuna/utils/merge_db.py

```diff
@@ -321,6 +331,12 @@ def write_merge_results(master_list, final_file, copy_files):
 
 def merge_text_file(master_file, copy_only, keep_keys, target_file=None):
   """merge db text files"""
+
+  if target_file is None:
+    raise ValueError(f'Invalid key value: {target_file}')
+  if not target_file and target_file.strip():
+    raise ValueError(f'Invalid key value: {target_file}')
```

Comment:
This conditional is problematic.
if not target_file, then target_file is None, False, 0, "", or empty set
if not target_file, then target_file.strip() will be evaluated, which will cause a separate kind of error on every possible value other than "" on which it will have no affect.
Perhaps the objective was to assert that the target_file is a non empty string or perhaps an existing file.

Reply:
checking target_file , already exist is not part of UT. test targets for valid i/p type